### ISSUE TITLE
[Obsidian] ...

### ISSUE THREAD

**Issue by smirnovmaksim91** [raycast/extensions#4092]
### Extension

https://www.raycast.com/marcjulian/obsidian

#### Error:
```
 Error: Worker terminated due to reaching memory limit: JS heap out of memory

NodeError:errors:393:5
[kOnExit]:worker:277:26
Worker.<computed>.onexit:worker:199:20
```

### Steps To Reproduce

search notes

### Current Behaviour

the search window opens

### Expected Behaviour

error

**Comment by smirnovmaksim91** [comment]
I have about 3,000 notes in my vault. I tried reinstalling the extension, it doesn't help.

**Comment by marcjulianschwarz** [comment]
It should work fine with this amount of notes. Do you have lots of images or other files in your vault?

**Comment by smirnovmaksim91** [comment]
I have about 1,500 images in storage

**Comment by marcjulianschwarz** [comment]
Hmm, could you try to use the command without the images? Maybe create a new vault where you copy only the notes?

**Comment by chadchudson** [comment]
I'm getting this exact error, as well. Not using images.

**Comment by marcjulianschwarz** [comment]
Does it happen with all commands?

**Comment by chadchudson** [comment]
> 

No, everything works perfectly except for the menu bar - the menu bar icon turns into a danger icon and completely breaks at the first attempt to use it. 

Error: Worker terminated due to reaching memory limit: JS heap out of memory

NodeError:errors:393:5
[kOnExit]:worker:277:26
Worker.<computed>.onexit:worker:199:20

**Comment by marcjulianschwarz** [comment]
Does the starred notes command work?

**Comment by chadchudson** [comment]
> 

No, I get:
Error: Worker terminated due to reaching memory limit: JS heap out of memory

NodeError:errors:393:5
[kOnExit]:worker:277:26
Worker.<computed>.onexit:worker:199:20

**Comment by marcjulianschwarz** [comment]
So only starred notes command and menu bar are not working?

**Comment by thomaslombart** [comment]
Hey @ChadChudson @SmirnovMaksim91, we've fixed a performance issue in the 1.54.0 release that should make this type of bugs less likely to appear. Do you still have it?
